src/include/ndpi_includes.h

```diff
@@ -57,7 +57,7 @@
 #if defined __NetBSD__ || defined __OpenBSD__
 #include <netinet/in_systm.h>
 
-#ifdef __OpenBSD__
```

Comment:
Is `#ifdef` not supported by OpenBSD's CPP?

Reply:
Oh, this is supported. I think I made it by mistake.

Reply:
@adek05 Can you please update this PR so I can merge it?

Reply:
@lucaderi I will take over this PR ASAP.